src/tainting/Taint.mli

```diff
@@ -231,3 +231,4 @@ type signature = Findings.t
  *)
 
 val _show_finding : finding -> string
+val _show_signature : signature -> string
```

Comment:
Typically we use leading underscores only for symbols that are not used or exposed publicly. I get that you're just following what's already here in this case, but do you know why this was done here?

Reply:
These are just exposed for debugging according to a comment on the previous comment. I will follow this up with renaming all of these to be `debug_show_*` per that comment.